promql/lex.go

```diff
@@ -127,6 +127,7 @@ const (
 	itemDIV
 	itemLAND
 	itemLOR
+	itemLUnless
```

Comment:
Yes, leaving the 3 letter abbreviations uppercase makes sense to me. Let me know whether you think it should be done in this PR or if a follow-up pr is sufficient.